src/Http/Controllers/Admin/Menu/ItemController.php

```diff
@@ -18,7 +18,8 @@ public function setup()
         $menuId = Request::input('menu');
 
         $this->crud->setModel(Item::class);
-        $this->crud->setRoute(route('crud.item.index', ['menu' => $menuId]));
+        $this->crud->setRoute(route('crud.item.index'));
+        $this->crud->setIndexRoute('crud.item.index', ['menu' => (int) request('menu')]);
```

Comment:
Why don't you use the $menuId variable defined above?

Reply:
Fixed